sections/Discussion.tex

```diff
@@ -1 +1,46 @@
+}
+
+Overall, we see that there is a large amount of overlap between the models, which is interesting since models use different meta-information to create the various topic distributions.
+This indicates that the models share many of the top words, and indicate a slight deviation between the models due to the meta-information.
+The Doc-topic model shows words, like "farming" and "production", which is what the article is mostly about.
+This behavior is to be expected since the performance of \gls{lda} has been explored and evaluated before. 
+Author-topic specific words are not very present and are only showing three unique words: "byder", "hører" and "børnene".
+This indicates that the Author-topic model has trouble generalizing what the author of this article(Peter Tordrup Larsen) is writing about since then he has written $5002$ articles. 

```

Comment:
```suggestion
This indicates that the Author-topic model has trouble generalizing what the author of this article (Peter Tordrup Larsen) is writing about, possibly because he has written $5002$ articles in our dataset. 
```